test/test.sh

```diff
@@ -32,5 +32,3 @@ for pcap in test/pcap/*.pcap; do
         rm -rf "${TMP}"
     fi
 done
-
-exit 0
```

Comment:
this is just common good practise when using set -eu; the script finished successfully thus lets exit it appropriately. It also prevents nasty hacks like 'echo "rm -rf /" >> script.sh

Reply:
why is this good practice? using -e will not allow this command to ever be reached if a non-zero code is discovered making this command 100% no-op

Reply:
uhm, no :p 
if all commands have run successfully, this will cause the script to cleanly access, so it's 100% of the time executed, when all tests pass. It basically says 'script is done, we can exit now'.

If the script fails, this will never be reached of course.

In scripts where set -e is not done, what could happen is, that the last command generated an error, and without an exit, the script would then now 'exit' with the last exit code, thus failure.

For example:
```sh
set -eu
lots_of_functions_here

# Script is done, print some pretty text to inform the user, if a output_device is conencted
set +e
echo "Test finished running" > /dev/output_device_that_is_not_always_here_and_thats_okay
```
While not the greatest example, here, without the exit 0, this script will always return a failure if there was no output_device, but that's okay. the script ran its course and there where no errors.

It's like cleaning up after one's self; we are done, lets inform the reader and executor that we are done

Reply:
we have `set -e` set, correct?  in this case `exit 0` at the end of the file becomes a no-op